ZaneMeroff/DDXRentals
# Create small listing card

### What's this PR do?
Add small listing card component.
### Where should the reviewer start?
Run web app and load up small listing card component.
### How should this be manually tested?
Load up small listing card and verify size and color theme matches and displays properly.
### Potential Changes / Next steps?
Need to implement with live data from fetch call.
### Relevant Tickets?
Display condensed view of specific listing
